modules/bullet/rigid_body_bullet.h

```diff
@@ -227,7 +227,7 @@ class RigidBodyBullet : public RigidCollisionObjectBullet {
 
 	void init_kinematic_utilities();
 	void destroy_kinematic_utilities();
-	_FORCE_INLINE_ KinematicUtilities *get_kinematic_utilities() const { return kinematic_utilities; }
+	_FORCE_INLINE_ class KinematicUtilities *get_kinematic_utilities() const { return kinematic_utilities; }
```

Comment:
Why do you added a class here?
KinematicUtilities is a struct and also is defined in this file